docs/host-list.rst

```diff
@@ -79,10 +79,10 @@ server. Any modern hardware should work 100% fine.
 Free hosting
 ------------
 
-`Google Cloud Compute Free Tier <https://cloud.google.com/free/docs/gcp-free-tier>`_,
-`Oracle Cloud Compute Always Free <https://oracle.com/cloud/free/#always-free>`_ and
-`AWS EC2 Free Tier <https://aws.amazon.com/free/>`_ have free tier VPSes suitable for small bots.
-AWS EC2 is not *always* free—it's a 12 month free trial.
+`Google Cloud Compute Engine <https://cloud.google.com/free/docs/gcp-free-tier>`_,
+`Oracle Cloud Compute <https://oracle.com/cloud/free/#always-free>`_ and
+`AWS EC2 <https://aws.amazon.com/free/>`_ have free tier VPSes suitable for small bots.
+Note: AWS EC2's free tier does not last forever - it's a 12 month trial.
 Additionally, new Google Cloud customers get a $300 credit which is valid
 for 12 months.
```

Comment:
It doesn't look that good without any line breaks:
![image](https://user-images.githubusercontent.com/6032823/86947970-d137ed00-c14c-11ea-9cd3-eff76483538d.png)

Perhaps we could add some?
![image](https://user-images.githubusercontent.com/6032823/86947926-c3826780-c14c-11ea-9688-e5053a239b4c.png)

```suggestion
| `Google Cloud Compute Engine <https://cloud.google.com/free/docs/gcp-free-tier>`_,
  `Oracle Cloud Compute <https://oracle.com/cloud/free/#always-free>`_ and
  `AWS EC2 <https://aws.amazon.com/free/>`_ have free tier VPSes suitable for small bots.
| **Note:** AWS EC2's free tier does not last forever - it's a 12 month trial.
| Additionally, new Google Cloud customers get a $300 credit which is valid
for 12 months.
```